packages/create-svelte/rollup.config.js

```diff
@@ -5,8 +5,8 @@ import json from '@rollup/plugin-json';
 export default {
 	input: 'cli/index.js',
 	output: {
-		file: 'bin',
-		format: 'cjs',
+		file: 'bin.js',
+		format: 'esm',
```

Comment:
Why change the format? Is this even related to the changes in this PR?

Reply:
there's just no reason to use CJS 🤷

Reply:
That's true, it just threw me off a little that this change is part of the PR 😄